source/fundamentals/connection/connect.txt

```diff
@@ -87,19 +87,30 @@ Other Connection Targets
 Connect to Atlas
 ~~~~~~~~~~~~~~~~
 
-In the following example, the driver uses a sample connection URI to
-connect to a MongoDB instance on Atlas, using placeholders for your
-username and password. In this example, the client uses
-the :ref:`Stable API <csharp-stable-api>` feature, though
-you can connect to Atlas without this feature.
+To connect to a MongoDB deployment on Atlas, you need to create a client. You can
```

Comment:
Suggestion:

No changes necessary necessary, but technically, I think you can connect without creating a client. You can use some other tool or application in which a client is already created for you. Instead, this sentence could suggest the path of action using the following language:

```suggestion
To connect to a MongoDB deployment on Atlas, create a client. You can
```